servers/text_server.h

```diff
@@ -59,7 +59,8 @@ class TextServer : public Object {
 		JUSTIFICATION_KASHIDA = 1 << 0,
 		JUSTIFICATION_WORD_BOUND = 1 << 1,
 		JUSTIFICATION_TRIM_EDGE_SPACES = 1 << 2,
-		JUSTIFICATION_AFTER_LAST_TAB = 1 << 3
+		JUSTIFICATION_AFTER_LAST_TAB = 1 << 3,
+		JUSTIFICATION_CONSTRAIN_ELLIPSIS = 1 << 4
```

Comment:
See https://github.com/godotengine/godot/pull/51178/files#r685784120, I would add a trailing comma here and below to avoid line diffs on unrelated enum values when adding elements.

Reply:
Done (for JustificationFlag, LineBreakFlag and TextOverrunFlag).